experiment/run_experiment.py

```diff
@@ -448,28 +451,26 @@ def main():
                                required=False,
                                default=None,
                                choices=all_fuzzers)
-    fuzzers_group.add_argument('-cf',
-                               '--changed-fuzzers',
-                               help=('Use fuzzers that have changed since the '
-                                     'last experiment. The last experiment is '
-                                     'determined by the database your '
-                                     'experiment uses, not necessarily the '
-                                     'fuzzbench service'),
-                               action='store_true',
-                               required=False)
+    fuzzers_group.add_argument('-s',
+                               '--no-seeds',
+                               help='Should trials be conducted without seed corpora.',
+                               required=False,
+                               default=False,
+                               action='store_true')
+    fuzzers_group.add_argument('-d',
```

Comment:
-nd ?

Reply:
I don't like doing two char short flags. I think it's more normal to use an unrelated letter than to use a char short flag (like how `-d` in AFL means deterministic so the flag for dictionary is `-x`)

Reply:
Here flag means the opposite, so it is very confusing. -d mean no dict is so confusing.

Reply:
Changed it.